bin/webpack-dev-server.js

```diff
@@ -247,7 +247,7 @@ function processOptions(wpOpt) {
 	new Server(webpack(wpOpt), options).listen(options.port, options.host, function(err) {
 		if(err) throw err;
 
-		var uri = protocol + "://" + options.host + ":" + options.port + "/";
+		var uri = " " + protocol + "://" + options.host + ":" + options.port + "/";
```

Comment:
The `uri` var is also used if `options.open` is given, and that would break. I think doing `console.log(" " + uri)` would make a lot more sense.